sdks/go/pkg/beam/core/runtime/xlangx/expansionx/process_test.go

```diff
@@ -22,15 +22,28 @@ import (
 	"testing"
 )
 
+func TestFindOpenPort(t *testing.T) {
+	port, err := findOpenPort()
+	if err != nil {
+		t.Fatalf("failed to find open port, got %v", port)
+	}
+	if port < 1 || port > 65535 {
+		t.Errorf("port out of TCP range [1, 66535], got %d", port)
+	}
+}
+
 func TestNewExpansionServiceRunner(t *testing.T) {
 	testPath := "path/to/jar"
 	testPort := "8097"
-	serviceRunner := NewExpansionServiceRunner(testPath, testPort)
+	serviceRunner, err := NewExpansionServiceRunner(testPath, testPort)
+	if err != nil {
+		t.Fatalf("NewExpansionServiceRunner failed, got %v", err)
+	}
 	if serviceRunner.jarPath != testPath {
 		t.Errorf("JAR path mismatch: wanted %v, got %v", testPath, serviceRunner.jarPath)
 	}
 	if serviceRunner.servicePort != testPort {
-		t.Errorf("service port mismatch: wanted %v, got %v", testPort, serviceRunner.servicePort)
+		t.Errorf("service port mismatch: wanted %v, got %v", testPort, testPort)
```

Comment:
This is an example of where using the got, want convention helps avoid copypasta mistakes like duplicating what we want, and what we got.

Doing something like

```
if got, want := serviceRunner.servicePort, testPort; got != want {
		t.Errorf("service port mismatch: got %v, want %v", got, want)
}
```

Similar elsewhere. I noticed this last time after I merged it, so now's a good time to fix the older test cases.

Reply:
Changed.